corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -447,6 +468,14 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             FormplayerFrontend.trigger("menu:search", searchText);
         },
 
+        searchMoreAction: function () {
+            if (!$('#sidebar-region').hasClass('in')) {
+                $([document.documentElement, document.body]).animate({
+                    scrollTop: $('#content-container').offset().top - constants.BREADCRUMB_HEIGHT_PX,
+                }, 350);
```

Comment:
I'm not familiar with this functionality-- what exactly is this doing?

Reply:
This is using jquery to smoothly scroll the top of the page to the top of the search filters, in this case defined as the top of the `#content-container` minus the breadcrumb height, so it's offset by that ~46px. `350` how long it takes in ms, which matches bootstrap's built-in animation for "collapse".

Reply:
ah neat